### PR TITLE
BZ#1868285 Expand on release extraction steps

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1868285

### CODE REVIEW

**Comment by codyhoag** [comment]
@gpei PTAL. Can you also confirm this affects OCP 4.4 and later? Thanks!

Preview: https://restricted-network-fix--ocpdocs.netlify.app/openshift-enterprise/latest/installing/install_config/installing-restricted-networks-preparations.html#installation-mirror-repository_installing-restricted-networks-preparations

**Comment by shellyyang1989** [comment]
LGTM. 
Yes. It affects 4.4 and later.

File: modules/installation-mirror-repository.adoc [hunk 1]
```diff
@@ -148,21 +148,27 @@ the `imageContentSources` data that you require when you install your cluster.
...
-
++
 [NOTE]
 ====
 The image name gets patched to Quay.io during the mirroring process, and the podman images will show Quay.io in the registry on the bootstrap virtual machine.
 ====
 
 . To create the installation program that is based on the content that you
 mirrored, extract it and pin it to the release:
+** If your mirror host does not have internet access, run the following command:
```
Comment (ahardin-rh) [inline modules/installation-mirror-repository.adoc:159]: It looks like Internet should be capitalized here according to the IBM Style Guide, though I see a lot of inconsistencies across out repo. I can create a separate PR to fix incorrect instances.

**Comment by ahardin-rh** [comment]
Just one comment. Otherwise, LGTM!

File: modules/installation-mirror-repository.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/installation-mirror-repository.adoc:159]: Good call! I just learned about that ISG rule the yesterday; I need to go back through all my pre-yesterday PRs and update :sweat_smile:. Thanks!

**Pull request merged**